test/functional/cmdLineTests/cmdLineTest_J9tests/j9tests_consoleUpTo17.xml

```diff
@@ -54,4 +54,12 @@
     <command>$EXE$ $OPENS$ $CP$ -D$DEFAULT_ENCODING_PROP$=$ENCODING2$ $TARGET$ err $ENCODING2$</command>
     <return type="success" value="0" />
   </test>
+
+  <test id="With -Dfile.encoding=CP943C default Charset is x-IBM943C">
+    <command>$EXE$ -Dfile.encoding=CP943C -cp $Q$$JARPATH$$Q$ DefaultCharset</command>
```

Comment:
Why is `CP943C` a good choice for this test? Is the risk low that this "historical" name will be removed (see [RegisteredCharsets.java](https://github.com/ibmruntimes/openj9-openjdk-jdk/blob/9b3a244652524d4dfcf7581dfd5a080816361ca5/test/jdk/java/nio/charset/Charset/RegisteredCharsets.java#L575))?

Reply:
CP943C is the charset called out in the issue. Any charset in jdk.charsets would do. Existing versions won't change to remove it, in the future anything can happen. If the charset isn't available in a future version we can update the test. The charset is still available in the head stream.

Reply:
Fair enough.